common/main/root.xml

```diff
@@ -5596,7 +5596,6 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 			<nameField type="given">∅∅∅</nameField>
 			<nameField type="given-informal">∅∅∅</nameField>
 			<nameField type="given2">∅∅∅</nameField>
-			<nameField type="surname">∅∅∅</nameField>
```

Comment:
Should we leave the surname field in root?

Reply:
No, let's remove there.